pkg/cloud/services/networking/network.go

```diff
@@ -128,13 +128,12 @@ func (s *Service) ReconcileNetwork(clusterName string, openStackCluster *infrav1
 	}
 	record.Eventf(openStackCluster, "SuccessfulCreateNetwork", "Created network %s with id %s", networkName, network.ID)
 
-	_, err = attributestags.ReplaceAll(s.client, "networks", network.ID, attributestags.ReplaceAllOpts{
-		Tags: []string{
-			"cluster-api-provider-openstack",
-			clusterName,
-		}}).Extract()
-	if err != nil {
-		return err
+	if len(openStackCluster.Spec.Tags) > 0 {
+		_, err = attributestags.ReplaceAll(s.client, "networks", network.ID, attributestags.ReplaceAllOpts{
+			Tags: openStackCluster.Spec.Tags}).Extract()
+		if err != nil {
+			return err
+		}
```

Comment:
Since Tags is added to type.go, we need to reflect it to openStackCluster.Status.Network, openStackCluster.Status.Network.Subnet respectively.

Reply:
no, I need remove that part, because tag is given by `Spec`, no need to keep in `Status`..

Reply:
Since `Status` is the current status of the resources, it not odd that `openStackCluster.Status.Network` and `openStackCluster.Status.Network.Subnet` have tags.

Reply:
don't fully understand the comment, can you help to elaborate? thanks

Reply:
I understand I can not add tags to Status as I expected. The followings are I tried.
At first I tried adding Tags with omitempty in types.go, but never tags are added to Status. Maybe the reason is  https://github.com/kubernetes-sigs/kubebuilder/issues/521.
Second, If added Tags without omitempty in types.go, tags with empty string to Status is not permitted and ended with error.